test/Auto/auto_test.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+source ./shared-queens.rc
+sudo apt install python-dev python-pip
+sudo pip install --upgrade pip
+sudo pip install --ignore-installed python-openstackclient
+
+openstack server create --flavor m1.large --image $TEST_IMAGE --nic net-id=e12b30b7-d0d1-47f2-9be6-346913749ebc --key-name zz test_instance_$name --config-drive True
```

Comment:
Network id value need to be set in Travis

Reply:
fixed.